### ISSUE TITLE
Rating accessibility issues

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1615]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Incorrectly says to use up or down arrows to change
Expect: Should say use left or right arrows to change.

2. Typing a number should work, and we need to be sure that when focus is on the control, that it tells how many stars there are.

**Comment by cschleiden** [comment]
@betrue-final-final 
1) Up/down does work? 

2) Why is that required for the control? For a range of typically 1-5 stars, why do we need number input?

**Comment by betrue-final-final** [comment]
Up and down just jump from 1 to 5. To get any other values you need to go left and right, so that should be in the instructions.

As for numbers, sensible shortcuts are always welcome for blind users and people with manual issues. It's not a ship stopper, but would enhance quality.

________________________________

**Comment by cschleiden** [comment]
1) No? Just tried it on IE11, Edge, and Chrome.

**Comment by betrue-final-final** [comment]
Did you have JAWS on?

________________________________

**Comment by cschleiden** [comment]
No.. does it change the interaction with the page? :/

**Comment by betrue-final-final** [comment]
It often affects keyboarding. Without JAWS, keyboarding could be great, but turn it on and things sometimes change.

________________________________

**Comment by betrue-final-final** [comment]
And you need to use JAWS on IE.

________________________________

**Comment by betrue-final-final** [comment]
Fyi, JAWS on IE is the overwhelming majority environment for blind users.

________________________________

**Comment by cschleiden** [comment]
@betrue-final-final Oh okay, I did not know that. I'll give it a try

**Comment by betrue-final-final** [comment]
Ok, let's build this against Narrator + Edge. It appears that when the rating is changed, it just says "Selected", when it should also say the rating, such as "3 out of 5 stars"

**Comment by betrue-final-final** [comment]
Any non- ARIA 1.0 standard JAWS behavior we will send to them for fixing.

**Comment by betrue-final-final** [comment]
Ok, I was able to refresh, and it does report the rating change. No helper text though in Narrator.

**Comment by betrue-final-final** [comment]
This is no longer reproing.
